pkg/services/searchusers/searchusers.go

```diff
@@ -43,6 +45,10 @@ func ProvideUsersService(searchUserFilter user.SearchUserFilter, userService use
 func (s *OSSService) SearchUsers(c *contextmodel.ReqContext) response.Response {
 	result, err := s.SearchUser(c)
 	if err != nil {
+		grafanaErr := errutil.Error{}
+		if !errors.As(err, &grafanaErr) {
+			return response.Err(err)
+		}
 		return response.Error(500, "Failed to fetch users", err)
```

Comment:
i am a bit unsure if we want to add all Errors for this. I know we discussed that we should omit user information in terms of "Brute Force" or more specifically "ID Enumeration". @mgyongyosi , I know you mentioned this?

Could we see if the error is of "ErrSorting" then we could return the error, otherwise we should omit the error